spyder/plugins/ipythonconsole/widgets/debugging.py

```diff
@@ -82,6 +110,21 @@ def _handle_input_request(self, msg):
         # Make sure that all output from the SUB channel has been processed
         # before entering readline mode.
         self.kernel_client.iopub_channel.flush()
+        self._input_ready = True
+        if not self.is_debugging():
+            return super(DebuggingWidget, self)._handle_input_request(msg)
+
+        # While the widget thinks only one input is going on,
+        # Other functions are sending messages to the kernel.
+        # This must be properly processed to avoid dropping messages.
+        # If the kernel was not ready, the messages are queued.
+        # Send the next one
```

Comment:
This last line doesn't make sense to me. What do you mean by

    Send to next one

?